src/commands/cli/install/test.ts

```diff
@@ -417,7 +371,7 @@ export default class Test extends SfdxCommand {
     }),
     channel: flags.string({
       description: messages.getMessage('channelFlag'),
-      options: Object.values(Channel),
+      options: [Channel.STABLE, Channel.STABLE_RC],
```

Comment:
This file had its own Channel enum that only specified `stable` and `stable-rc`, so limiting choices to the same